android/java/org/chromium/chrome/browser/infobar/BraveInfoBarIdentifier.java

```diff
@@ -22,7 +21,7 @@
 public @interface BraveInfoBarIdentifier {
     int INVALID = -1;
     int BRAVE_CONFIRM_P3A_INFOBAR_DELEGATE = 500;
-    int WAYBACK_MACHINE_INFOBAR_DELEGATE = 502;
+    // int WAYBACK_MACHINE_INFOBAR_DELEGATE = 502; - deprecated
```

Comment:
@AlexeyBarabash Can you check this change?
AFAIK, android doesn't use wayback machine. so I thought just deleting this seems safe.

Reply:
@simonhong yes, it can be deleted safely. The wayback machine is not used on Android.